ark/system/simulation/simulator_node.py

```diff
@@ -56,9 +57,14 @@ def __init__(self, global_config):
         # Setup backend
         self.backend_type = self.global_config["simulator"]["backend_type"]
         if self.backend_type == "pybullet":
+            from ark.system.pybullet.pybullet_backend import PyBulletBackend
             self.backend = PyBulletBackend(self.global_config)
         elif self.backend_type == "mujoco":
+            from ark.system.mujoco.mujoco_backend import MujocoBackend
             self.backend = MujocoBackend(self.global_config)
+        elif self.backend_type == "genesis":
+            from ark.system.genesis.genesis_backend import GenesisBackend
+            self.backend = GenesisBackend(self.global_config)
         else:
```

Comment:
i think a registry, eg [here](https://github.com/Robotics-Ark/ark_ml/blob/5bd3554be56aa1296f59b5fbb91704078bff44e9/arkml/core/registry.py#L1-L17) would be a good idea to implement instead of this list of if-statements, but that could be another PR, please create an issue and put the link as a reply here